web/src/unread_ui.ts

```diff
@@ -1,25 +1,29 @@
 import $ from "jquery";
+import assert from "minimalistic-assert";
 
 import render_mark_as_read_disabled_banner from "../templates/unread_banner/mark_as_read_disabled_banner.hbs";
 import render_mark_as_read_only_in_conversation_view from "../templates/unread_banner/mark_as_read_only_in_conversation_view.hbs";
 import render_mark_as_read_turned_off_banner from "../templates/unread_banner/mark_as_read_turned_off_banner.hbs";
 
 import * as message_lists from "./message_lists";
+import type {Message} from "./message_store";
 import * as narrow_state from "./narrow_state";
 import {page_params} from "./page_params";
 import {web_mark_read_on_scroll_policy_values} from "./settings_config";
 import * as unread from "./unread";
+import type {FullUnreadCountsData} from "./unread";
 import {user_settings} from "./user_settings";
 
 let user_closed_unread_banner = false;
 
-const update_unread_counts_hooks = [];
+type UnreadCountHook = (counts: FullUnreadCountsData, skip_animations: boolean) => void;
```

Comment:
For consistency this should be plural: `UnreadCountsHook` or `UpdateUnreadCountsHook`.

Reply:
Updated